tests/monitor/docker-compose.test.yml

```diff
@@ -13,7 +13,8 @@ services:
       PROCESS_CHECKPOINT_TRUSTED_OWNERS: ${PROCESS_CHECKPOINT_TRUSTED_OWNERS:-fcoN_xJeisVsPXA-trzVAuIiqO3ydLQxM-L4XbrQKzY,-HFe6PleLxj1EdFMYMSetT2NIJioDsZIktn-Y0AwP54,WjnS-s03HWsDSdMnyTdzB1eHZB2QheUWP_FVRVYxkXk}
       GRAPHQL_URL: ${GRAPHQL_URL:-https://arweave-search.goldsky.com/graphql}
       CHECKPOINT_GRAPHQL_URL: ${CHECKPOINT_GRAPHQL_URL:-https://arweave-search.goldsky.com/graphql}
+      ARWEAVE_URL: ${ARWEAVE_URL:-https://arweave.net}
       CHECKPONT_VALIDATION_THRESH: ${CHECKPONT_VALIDATION_THRESH:-1}
-      CHECKPONT_VALIDATION_STEPS: ${CHECKPONT_VALIDATION_STEPS:-0}
+      CHECKPONT_VALIDATION_STEPS: ${CHECKPONT_VALIDATION_STEPS:-51}
```

Comment:
This allows us to log the checkpoint being used in the cu but will still fall back to the latest checkpoint